codex/manifest/coders.nim

```diff
@@ -145,6 +162,15 @@ proc decode*(_: type Manifest, data: openArray[byte]): ?!Manifest =
   if pbHeader.getField(7, pbErasureInfo).isErr:
     return failure("Unable to decode `erasureInfo` from manifest!")
 
+  if pbHeader.getField(8, filename).isErr:
+    return failure("Unable to decode `filename` from manifest!")
+
+  if pbHeader.getField(9, mimetype).isErr:
+    return failure("Unable to decode `mimetype` from manifest!")
+
+  if pbHeader.getField(10, uploadedAt).isErr:
+    return failure("Unable to decode `uploadedAt` from manifest!")
+
```

Comment:
@gmega I tried to decode the metadata without encoding empty values when the metadata doesn't exist, and it seems that protobuf is fine with that. If it were to raise an error, the test inside testmanifest.nim would not pass when a manifest is created without metadata (see the 'Should encode/decode to/from base manifest' test).